api/test.php

```diff
@@ -0,0 +1,179 @@
+<?php
+/*
+    Copyright (C) 2004-2010 Kestas J. Kuliukas / Timothy Jones
+
+	This file is part of webDiplomacy.
+
+    webDiplomacy is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    webDiplomacy is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with webDiplomacy.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace webdiplomacy_api;
+
+/**
+ * Simple class to test HTPP calls on WebDiplomacy API located at `/api.php`.
+ * @package webdiplomacy_api
+ */
+class ApiCall {
+	/**
+	 * Available API calls.
+	 * @var array
+	*/
+	static private $routes = array(
+		'players/cd' => 'GET',
+		'players/missing_orders' => 'GET',
+		'game/status' => 'GET',
+		'game/orders' => 'JSON'
+	);
+
+	/**
+	 * URL of WebDiplomacy instance (e.g. `http://mysite.com` ).
+	 * API access will be "$siteUrl/api.php" (e.g. `http://mysite.com/api.php` ).
+	 * @var string
+	 */
+	private $siteUrl;
+
+	/**
+	 * API access key to be used to make API calls.
+	 * @var string
+	 */
+	private $apiKey;
+
+	/**
+	 * Initialize an ApiCall object.
+	 * @param string $siteUrl - URL of a WebDiplomacy instance.
+	 * @param string $apiKey - API access key/
+	 */
+	public function __construct($siteUrl, $apiKey) {
+		$this->siteUrl = trim($siteUrl, " /\t\n\r\0\x0B");
```

Comment:
Can you add a comment for what this does? It looks like the kind of thing that might be different from webserver to webserver. Not the end of the world, but it looks .. delicate

Reply:
(For test code I’m not too bothered, but I saw this above I think)